pkg/util/net/http.go

```diff
@@ -67,6 +70,9 @@ func SetTransportDefaults(t *http.Transport) *http.Transport {
 	if t.TLSHandshakeTimeout == 0 {
 		t.TLSHandshakeTimeout = defaultTransport.TLSHandshakeTimeout
 	}
+	if err := http2.ConfigureTransport(t); err != nil {
```

Comment:
Yep. And if you check the duped one https://github.com/golang/go/issues/14391, the fix made it in to 1.6.2. So do we have to do this even with 1.6.2?